### PR TITLE
Added rounded corners and fixed aspect ratio for carousel elements

### PR DESCRIPTION
**Pull request by nrleuenb**
All carousel elements are now constrained to fit within a 200px by 200px box. 

The branch discovery-ui contains more improvements to the carousel elements, including spacing fixes and a border.

Also added spaces before brackets in `discovery.scss` for consistency

### CODE REVIEW

**Comment by nrleuenb** [comment]
@tiffanyperumpail Can you look this over?

File: app/assets/stylesheets/discovery.scss [hunk 1]
```diff
@@ -1,37 +1,46 @@
...
 .image {
-  width: 100%;
+  width: scale;
   height: scale;
   object-fit: contain;
+  max-width: 200px;
+  max-height: 200px;
+  border-radius: 15px;
+  border-color: white;
+  border-width: 5px;
```
Comment (tiffanyperumpail) [inline app/assets/stylesheets/discovery.scss]: Looks good.

**Review by tiffperumpail — APPROVED** [review]
The CSS rules are organized. Tried it out locally and the carousel elements look good.

**Comment by Juggernaut265** [comment]
The new changes look fine, but this branch should not be merged until the conflicts are fixed and pushed. 

Edit: I fixed this and commited.

File: app/views/discovery/_carousel_vendor_element.html.haml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 %div{:class => "container carousel_element #{local_assigns[:ownership].gsub(" ", "_")}", :id => local_assigns[:vendor].name.gsub(" ", "_"), :onclick => "routeToPageDirectly('#{vendor_path(local_assigns[:vendor].id)}')"}
```
Comment (nrleuenb) [inline app/views/discovery/_carousel_vendor_element.html.haml]: @Juggernaut265 shouldn't this be the same as products?

**Comment by nrleuenb** [comment]
@Juggernaut265 fixed merge conflicts

File: app/views/discovery/_carousel_vendor_element.html.haml (hunk 2 above)
Comment (nrleuenb) [reply app/views/discovery/_carousel_vendor_element.html.haml]: Which one is correct? Should I revert my fixes?

Comment (Juggernaut265) [reply app/views/discovery/_carousel_vendor_element.html.haml]: I thought the "carousel_image.button" was your change, so I got rid of the button. We should keep the one that you originally wrote.

**Comment by dwang733** [comment]
LGTM, carousel elements definitely look better now!

**Pull request merged**